pkg/server/commands/list_objects_test.go

```diff
@@ -140,7 +140,7 @@ func TestListObjectsDispatchCount(t *testing.T) {
 				define member: [user, group#member]
 			`,
 			tuples: []*openfgav1.TupleKey{
-				tuple.NewTupleKey("group:eng#member", "member", "group:fga#member"),
+				tuple.NewTupleKey("group:eng", "member", "group:fga#member"),
```

Comment:
Why is this an invalid tuple? The model allows `group#member` to be related to `group#member`.

Reply:
Yes, but the `object` field cannot be `group:eng#member`
<img width="951" alt="image" src="https://github.com/openfga/openfga/assets/5374887/22cc6c87-6b81-44de-9a59-47df6e9239ae">